jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -335,7 +336,9 @@
                   (- (/ (m->mm (send bin-box :dimensions :x)) 2.0))
                   0.0 (/ (m->mm (send bin-box :dimensions :z)) 2.0))
                 offset) :world)
-      (send bin-coords :rotate (* pi (/ coords-pitch 180)) :y)
+      (send bin-coords :rotate (deg2rad (aref rpy 1)) :y)
+      (send bin-coords :rotate (deg2rad (aref rpy 2)) :z)
+      (send bin-coords :rotate (deg2rad (aref rpy 0)) :x)
```

Comment:
The order of `:rpy` in `make-coords` is `yaw-pitch-roll`
You can check this like below:
```
1.irteusgl$ setq a (make-cube 100 100 100)
#<body #X493f8e0 (:cube 100.0 100.0 100.0) 0.0 0.0 0.0 / 0.0 0.0 0.0>
2.irteusgl$ setq b (make-coords)
#<coordinates #X4adeb98  0.0 0.0 0.0 / 0.0 0.0 0.0>
3.irteusgl$ send b :scale 6
#<coordinates #X4adeb98  0.0 0.0 0.0 / 0.0 0.0 0.0>
4.irteusgl$ objects (list a b)
;; (make-irtviewer) executed
(#<body #X493f8e0 (:cube 100.0 100.0 100.0) 0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X4adeb98  0.0 0.0 0.0 / 0.0 0.0 0.0>)
```
![screenshot from 2017-05-20 11 23 16](https://cloud.githubusercontent.com/assets/14994939/26272336/9ef509d2-3d50-11e7-9987-a2d035456aa0.png)
```
5.irteusgl$ setq c (make-coords :rpy #f(1 0 0))
#<coordinates #X4ea4640  0.0 0.0 0.0 / 1.0 0.0 0.0>
6.irteusgl$ objects (list a b c)
(#<body #X493f8e0 (:cube 100.0 100.0 100.0) 0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X4adeb98  0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X4ea4640  0.0 0.0 0.0 / 1.0 0.0 0.0>)
7.irteusgl$ send c :scale 5
#<coordinates #X4ea4640  0.0 0.0 0.0 / 1.0 0.0 0.0>
```
![screenshot from 2017-05-20 11 24 53](https://cloud.githubusercontent.com/assets/14994939/26272337/a1008422-3d50-11e7-89f5-fc0d0b631420.png)
```
8.irteusgl$ setq c (make-coords :rpy #f(0 1 0))
#<coordinates #X4f62080  0.0 0.0 0.0 / 0.0 1.0 0.0>
9.irteusgl$ send c :scale 5
#<coordinates #X4f62080  0.0 0.0 0.0 / 0.0 1.0 0.0>
10.irteusgl$ objects (list a b c)
(#<body #X493f8e0 (:cube 100.0 100.0 100.0) 0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X4adeb98  0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X4f62080  0.0 0.0 0.0 / 0.0 1.0 0.0>)
```
![screenshot from 2017-05-20 11 25 59](https://cloud.githubusercontent.com/assets/14994939/26272338/a63bd446-3d50-11e7-8683-704e1b14d89c.png)
```
11.irteusgl$ setq c (make-coords :rpy #f(0 0 1))
#<coordinates #X504c798  0.0 0.0 0.0 / 0.0 0.0 1.0>
12.irteusgl$ send c :scale 5
#<coordinates #X504c798  0.0 0.0 0.0 / 0.0 0.0 1.0>
13.irteusgl$ objects (list a b c)
(#<body #X493f8e0 (:cube 100.0 100.0 100.0) 0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X4adeb98  0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X504c798  0.0 0.0 0.0 / 0.0 0.0 1.0>)
```
![screenshot from 2017-05-20 11 26 33](https://cloud.githubusercontent.com/assets/14994939/26272339/a808ab3c-3d50-11e7-9593-f04d5abb6beb.png)